### ISSUE TITLE
Provide system details in the Prerequisites step

### ISSUE THREAD

**Issue by acotiuga** [xwikisas/application-confluence-migrator#18]
In order to have a clear view of what is the current setup providing and what needs to be adjusted for the Confluence Migrator to properly work, some information can be obtained from several places. We should be able to get the current state and to recommend what to improve in:
* xwiki.cfg
* xwiki.properties
* JVM (Xms, Xmx, the memory of the machine)
* listeners (enable/disable)
* notifications

To cover all the cases we can use a Java helper that can use Configuration sources, can execute Linux commands, can check XWiki Objects..

**Comment by ldubost** [comment]
Yes, Ideally the prerequisite step should just verify any feature and provide a button to make the change
